tools/run_tests/run_interop_tests.py

```diff
@@ -1247,3 +1248,8 @@ def aggregate_http2_results(stdout):
       dockerjob.remove_image(image)
     else:
       print('Preserving docker image: %s' % image)
+
+if num_failures:
```

Comment:
I don't feel throwing exception is cleaner. sys.exit(1) is used when other types of jobs fail such as building docker image. This is consistent.